src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js

```diff
@@ -35,13 +38,24 @@ export class SetActions extends Component {
       return (
         <header className={styles.Actions}>
           <div className={styles.Left}>
-            <Search
-              className={cx(styles.Action, styles.Filter)}
+            <TextField
               name="filter"
-              value={this.props.filterTerm}
               placeholder="Filter Items"
-              onSubmit={this.props.onFilter}
```

Comment:
@agalin920 , @shrunyan im not 100% sure what this onSubmit is accomplishing or if I need to carry this over to MUI TextField ?